src/Hyperplane.jl

```diff
@@ -167,6 +170,9 @@ Otherwise this function throws an error.
                 elseif first_nonzero_entry_a == -1
                     factor = hp.a[i] / d[i]
                     first_nonzero_entry_a = i
+                    if halfspace && factor < 0
```

Comment:
Given `N` why not returning `convert(N, Inf)`?

Reply:
I created #750. To keep things consistent, let us do that change in one go.